main.ts

```diff
@@ -20,34 +26,84 @@ const DEFAULT_SETTINGS: PluginSettings = {
   lastUpdated: null,
   autoSync: true,
   highlightSortOldestToNewest: true,
+  syncNotesOnly: false,
   logFile: true,
   logFileName: 'Sync.md',
+  frontMatter: false,
+  frontMatterTemplate: `---
+id: {{ id }}
+updated: {{ updated }}
+title: {{ title }}
+author: {{ author }}
+---
+`,
+  headerTemplate: `
+%%
+ID: {{ id }}
+Updated: {{ updated }}
+%%
+
+![]( {{ cover_image_url }})
+
+# About
+Title: {{ title }}
+Authors: {{ authorStr }}
+Category: #{{ category }}
+{%- if tags %}
+Tags: {{ tags }}
+{%- endif %}
+Number of Highlights: =={{ num_highlights }}==
+Readwise URL: {{ highlights_url }}
+Source URL: {{ source_url }}

```

Comment:
Can we please hide Source URL if it's empty? This field seems to be present for article types, but not for books.
![about1](https://user-images.githubusercontent.com/4033621/149646055-d9e9df43-d3e3-40e2-9bdf-11da6236c724.png)